contributing/data-model-schema.rst

```diff
@@ -331,7 +331,7 @@ Documentation for the released schema must be generated from the
 this task, and requires the schema definitions to have been published online
 as described above. To build the generated documentation for a given release::
 
-  /Applications/oxygen/schemaDocumentationMac.sh http://www.openmicroscopy.org/Schemas/OME/$RELEASE/ome.xsd -cfg:components/specification/omeOxygenDocConfig.xml
+  /Applications/oxygen/schemaDocumentationMac.sh https://www.openmicroscopy.org/Schemas/OME/$RELEASE/ome.xsd -cfg:components/specification/omeOxygenDocConfig.xml
```

Comment:
I'm assuming that whatever this does can cope with https.

Reply:
it does